sklearn/manifold/_spectral_embedding.py

```diff
@@ -189,7 +189,8 @@ def spectral_embedding(
         of the lobpcg eigen vectors decomposition when `eigen_solver ==
         'amg'`, and for the K-Means initialization. Use an int to make
         the results deterministic across calls (See
-        :term:`Glossary <random_state>`).
+        :term:`Glossary <random_state>`). If None, the random number
+        generator is the ``RandomState`` instance used by ``np.random``.
```

Comment:
Should we make this explicit in the whole doc.
```suggestion
        :term:`Glossary <random_state>`). If None, the random number
        generator is the ``RandomState`` instance used by ``np.random``,
        that is ``np.random.default_rng()``.
```

This somewhat is a general remark whose scope is greater than this PR.

Any though, @glemaitre and @amueller?

Reply:
I would think that it could be best to state in the glossary that is the redirection of the `random_state` term. We can give more details there.